### DISCUSSION TITLE
.netcore3.1 and net6 cultureinfo

### DISCUSSION THREAD

**Discussion by mzy666888** [dotnet/ResXResourceManager#562] [Q&A]
**Important**
Read https://github.com/dotnet/ResXResourceManager/blob/master/Documentation/Topics/Troubleshooting.md first.

**Describe the bug**
on `netcoreapp3.1`  
```
CultureInfo.GetCultures(CultureTypes.AllCultures)
```
I can find the cultureinfo  'name `zh-CN`。

but use `net6` , I can't found the cultureinfo name `zh-CN`, instand of name `zh-Hans-CN`
![f982e52adaa99009dc0bf86659d685f](https://github.com/dotnet/ResXResourceManager/assets/2587129/12de6a61-a8e5-446d-bd64-d663a9c02d8f)

![image](https://github.com/dotnet/ResXResourceManager/assets/2587129/10241da3-39cc-44cc-b52e-3456f7295e5a)

on 'rexsmanager' window add language ,can't found the cultureinfo name `zh-Hans-CN`.
![image](https://github.com/dotnet/ResXResourceManager/assets/2587129/bff71bd7-6a52-4885-9d75-f82f56e290ac)

**To Reproduce**

**Expected behavior**

**Comment by tom-englert** [comment]
Visual Studio is running on NetFramework, not Core.

Do you really need different translations for both zh-Hans-CN and zh-Hans?

**Comment by mzy666888** [comment]
i need to comfirm the resource file name. if I use `zh-CN.resources` on platform netcoreapp3.1,   but i can't found `zh-CN` cultureinfo when I changed platform  net6

**Comment by tom-englert** [comment]
What about zh-Hans? That's on all platforms.

**Comment by tom-englert** [comment]
```
> var y = System.Globalization.CultureInfo.GetCultureInfo("zh-Hans-CN");
> var x = System.Globalization.CultureInfo.GetCultureInfo("zh-CN");
> WriteLine(x == y);
True
```
